File: hw/Vaca/u3/HW28SingletonAndAbstractFactory/HW28AbstractFactoryJavaScript/model/LinuxFactory.js
import GUIFactory from './GUIFactory.js';
import LinuxButton from './LinuxButton.js';
import LinuxMenu from './LinuxMenu.js';

class LinuxFactory extends GUIFactory {
    createButton() {
        return new LinuxButton();
    }

    createMenu() {
        return new LinuxMenu();
    }
}

export default LinuxFactory;

File: hw/ayuquina/u3/hw28DP-SingletonAndAbstractFactory/AbstractFactoryPattern/AbstractFactory/src/main/java/ec/edu/espe/abstractfactory/controller/LinuxFactory.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.espe.abstractfactory.controller;

/**
 *
 * @author Danny Ayuquina, LogicLegion, DCCO-ESPE
 */
public class LinuxFactory extends GUIFactory{

    @Override
    public Button createButton() {
        return new LinuxButton();
    }

    @Override
    public Menu createMenu() {
        return new LinuxMenu();
    }
    
}
